### ISSUE TITLE
rsb is broken with Travis CI build - get it working

### ISSUE THREAD

**Issue by noahgibbs** [noahgibbs/rsb#9]
Just turned it on. I should debug it.

Right now, in Ruby 2.0, it's broken because it's not installing the correct Bundler version. Config is always interesting to get right...

**Comment by eregon** [comment]
The Bundler version seems fine, from https://travis-ci.org/noahgibbs/rsb/jobs/601890280:
```
$ ruby --version
ruby 2.0.0p648 (2015-12-16 revision 53162) [x86_64-linux]
$ rvm --version
rvm 1.29.7 (latest) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]
$ bundle --version
Bundler version 1.16.6
$ gem --version
YAML safe loading is not available. Please upgrade psych to a version that supports safe loading (>= 2.0).
2.7.8
```

But `bundle install` fails:
```
$ bundle install
Warning: the running version of Bundler (1.16.6) is older than the version that created the lockfile (1.17.3). We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.
Fetching gem metadata from https://rubygems.org/.............
minitest-5.12.2 requires ruby version ~> 2.2, which is incompatible with the
current version, ruby 2.0.0p648
The command "bundle install" failed and exited with 5 during .
```
Probably needs an older `minitest` version.

**Comment by noahgibbs** [comment]
Thanks! I'm downgrading to 5.12.0 now, which doesn't seem to have that requirement. We'll see how it goes.

**Comment by noahgibbs** [comment]
Fixed by e320ffd7e585faf4f518db0ca1bacc8eb36727b0

**Comment by eregon** [comment]
Great, thanks!
